src/protect/mutation_translation.py

```diff
@@ -65,8 +65,29 @@ def run_transgene(job, snpeffed_file, rna_bam, univ_options, transgene_options):
                   '--rna_file', input_files['rna.bam'],
                   '--prefix', 'transgened',
                   '--pep_lens', '9,10,15']
-    docker_call(tool='transgene', tool_parameters=parameters, work_dir=work_dir,
-                dockerhub=univ_options['dockerhub'], tool_version=transgene_options['version'])
+
+    if fusion_calls:
+        fusion_files = {'fusion_calls': fusion_calls,
+                        'transcripts.fa.tar.gz': transgene_options['gencode_transcript_fasta'],
+                        'annotation.gtf.tar.gz': transgene_options['gencode_annotation_gtf'],
+                        'genome.fa.tar.gz': transgene_options['genome_fasta']}
+
+        fusion_files = get_files_from_filestore(job, fusion_files, work_dir, docker=False)
+        fusion_files['transcripts.fa'] = untargz(fusion_files['transcripts.fa.tar.gz'], work_dir)
+        fusion_files['genome.fa'] = untargz(fusion_files['genome.fa.tar.gz'], work_dir)
+        fusion_files['annotation.gtf'] = untargz(fusion_files['annotation.gtf.tar.gz'], work_dir)
+        fusion_files = {key: docker_path(path) for key, path in fusion_files.items()}
+        parameters += ['--transcripts', fusion_files['transcripts.fa'],
+                       '--fusions', fusion_files['fusion_calls'],
+                       '--genome', fusion_files['genome.fa'],
+                       '--annotation', fusion_files['annotation.gtf']]
+
+    docker_call(tool='transgene',
+                tool_parameters=parameters,
+                work_dir=work_dir,
+                dockerhub='jpfeil',
+                tool_version='latest')
```

Comment:
I'm using the latest version of Transgene, so commit 66ca1a1888eba7e6a5e8fa98a0bfc7f6ce72f4e1